packages/components/pie-notification/test/visual/pie-notification.spec.ts

```diff
@@ -120,201 +42,204 @@ test.describe('Props', () => {
                 // Arrange
-                await page.setViewportSize({ width: 375, height: 667 });
-                await mount(PieNotification, {
-                    props: {
-                        ...initialValues,
-                        leadingAction: mainAction,
-                    } as NotificationProps,
-                    slots: { default: slotContent },
-                });
+                const basePage = new BasePage(page, 'notification');
+                const props: Partial<NotificationProps> = {
+                    ...initialValues,
+                    leadingAction: mainAction,
+                };
 
+                await basePage.load({ ...props });
+
+                // Assert
                 await percySnapshot(page, 'PieNotification - leadingAction');
             });
 
-            test('should not render leadingAction when leadingAction not is provided', async ({ mount, page }) => {
+            test('should not render leadingAction when leadingAction not is provided', async ({ page }) => {
                 // Arrange
-                await page.setViewportSize({ width: 375, height: 667 });
-                await mount(PieNotification, {
-                    props: {
-                        ...initialValues,
-                        leadingAction: undefined,
-                    } as NotificationProps,
-                    slots: { default: slotContent },
-                });
+                const basePage = new BasePage(page, 'notification');
+                const props: Partial<NotificationProps> = {
+                    ...initialValues,
+                    leadingAction: undefined,
+                };
 
+                await basePage.load({ ...props });
+
+                // Assert
                 await percySnapshot(page, 'PieNotification - leadingAction not provided');
             });
         });
 
         test.describe('supportingAction', () => {
-            test('should not render supportingAction when leadingAction not is provided', async ({ mount, page }) => {
+            test('should not render supportingAction when leadingAction not is provided', async ({ page }) => {
                 // Arrange
-                await page.setViewportSize({ width: 375, height: 667 });
-                await mount(PieNotification, {
-                    props: {
-                        ...initialValues,
-                        leadingAction: undefined,
-                        supportingAction: secondaryAction,
-                    } as NotificationProps,
-                    slots: { default: slotContent },
-                });
+                const basePage = new BasePage(page, 'notification');
+                const props: Partial<NotificationProps> = {
+                    ...initialValues,
+                    leadingAction: undefined,
+                    supportingAction: secondaryAction,
+                };
```

Comment:
Same here, but rename to `supportingAction`, then update the following:

```suggestion
                const props: Partial<NotificationProps> = {
                    ...initialValues,
                    leadingAction: undefined,
                    supportingAction,
                };
```